docs/user-documentation/creating-collections-in-islandora-7.x-2.x.md

```diff
@@ -0,0 +1,27 @@
+# Creating Collections in Islandora 7.x-2.x
```

Comment:
Right now I've got list and grid Views of Basic Image content.  To create a View of a collection (filter the View) the child objects need to have some sort of reference to the parent object. So in the Basic Image content type form, the dropdown for selecting a parent collection should write the pcdm:memberOf  property to the object.  I've got #93 in and have added the suggestion re: pcdm:memberOf.  
If we had that (or some other reference to the parent) we'd be golden.

As an alternate if we had a solr field that included the parent resource, then that would probably work too. We don't have either at the moment.

Another potential method? would be to create a node View of the collection object itself and use/write the fedora:hasMember links in some manner.  We would need to have references to things like TN ... ideally in the fedora object (fedora object references the TN as external).